docs/api/field_types_reference/binaryfilefield.md

```diff
@@ -26,33 +26,30 @@ Note that both `BinaryFile` and `Media` Value and Type inherit from the `BinaryB
 |`mimeType`|string|The file's MIME type.|application/pdf|
 |`uri`|string|The binary file's `content/download` URI. If the URI doesn't include a host or protocol, it applies to the request domain.|/content/download/210/2707|
 |`downloadCount`|integer|Number of times the file was downloaded|0|
-|`path`|string|**deprecated**|N/A|
```

Comment:
This is also mentioned in 4.x branches. This is probably wrong there as well.

Reply:
`path` is `inputUri`according to https://github.com/ezsystems/ezplatform-kernel/blob/1.3/eZ/Publish/Core/FieldType/BinaryBase/Value.php#L97-L99